test/functionalTest/cases/4113_alarm_for_notification_queue_overpassing_a_given_threshold/alarm_for_notification_queue_overpassing_a_given_threshold.test

```diff
@@ -0,0 +1,230 @@
+echo '===================================================================='
+payload='{
+  "description": "serv3 subscriptions",
+  "subject": {
+    "entities": [
+      {
+        "id": "E",
+        "type": "T"
+      }
+    ]
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:'$LISTENER_PORT'/waitForever"
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions --payload "$payload" --tenant serv3
+echo
+echo
+
+
+echo "04. Create/update entity in serv1 3 times"
+echo "========================================="
+payload='{
+  "id": "E",
+  "type": "T",
+  "A": {
+     "value": 1,
+   "type": "Number"
+  }
+}'
+for i in {1..3}
+do
+   echo $i
+   orionCurl --url '/v2/entities?options=upsert,forcedUpdate' --payload "$payload" --tenant serv1 > /dev/null
+done
+echo
+echo
+
+
+echo "05. Create/update entity in serv2 4 times"
+echo "========================================="
+for i in {1..4}
+do
+   echo $i
+   orionCurl --url '/v2/entities?options=upsert,forcedUpdate' --payload "$payload" --tenant serv2 > /dev/null
+done
+echo
+echo
+
+
+echo "06. Create/update entity in serv3 5 times"
+echo "========================================="
+for i in {1..5}
+do
+   echo $i
+   orionCurl --url '/v2/entities?options=upsert,forcedUpdate' --payload "$payload" --tenant serv3 > /dev/null
+done
+echo
+echo
+
+
+echo "07. Grep log for notificationQueue alarm"
+echo "========================================"
+egrep 'Raising alarm|Releasing alarm' /tmp/contextBroker.log | awk -F'|' '{ print $10 }' | awk -F= '{ print $2 }'
+echo
+echo
+
+
+accumulatorStop $LISTENER_PORT
+
+
+--REGEXPECT--
+01. Subscribe serv1 to the accumulator endpoint that never responses
+====================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+02. Subscribe serv2 to the accumulator endpoint that never responses
+====================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+03. Subscribe serv3 to the accumulator endpoint that never responses
+====================================================================
+HTTP/1.1 201 Created
+Date: REGEX(.*)
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Content-Length: 0
+
+
+
+04. Create/update entity in serv1 3 times
+=========================================
+1
+2
+3
+
+
+05. Create/update entity in serv2 4 times
+=========================================
+1
+2
+3
+4
+
+
+06. Create/update entity in serv3 5 times
+=========================================
+1
+2
+3
+4
+5
+
+
+07. Grep log for notificationQueue alarm
+========================================
+Raising alarm NotificationQueue serv1: notification queue reached maximum threshold
```

Comment:
As far as I understand, we should also see here messages corresponding to serv2 and serv3 (and also default queue, when [this comment](https://github.com/telefonicaid/fiware-orion/pull/4375#pullrequestreview-1569179743) get addressed.

Reply:
Probably fixing https://github.com/telefonicaid/fiware-orion/pull/4375#discussion_r1288345726 and https://github.com/telefonicaid/fiware-orion/pull/4375#pullrequestreview-1569346634 helps with this.

Reply:
Fixed